tmt/steps/provision/mrack.py

```diff
@@ -204,11 +204,10 @@ class MrackHWDeviceElement(MrackHWElement):
     An element for device with op and description attributes
     """
 
-    def __init__(self, operator: str, description: str) -> None:
+    def __init__(self, **attrs: str) -> None:
         super().__init__('device')
 
-        # Use underscore prefix for attributes in mrack
-        self.attributes = {'_op': operator, '_description': description}
+        self.attributes = {f"_{key}": value for key, value in attrs.items()}
```

Comment:
It seems to me other classes could approach this in the same way, namely `MrackHWBinOp` and `MrackHWKeyValue`. I mean, there is some difference in treatment of how we use these three classes, we use keywords when creating `MrackHWDeviceElement` but positional parameters with `MrackHWKeyValue` and `MrackHWBinOp`. I'm afraid this change does not resolve this inconsistency, it just bends more code to support it. I would propose to nto do this change here and now, and if `**attrs` & invocations with keywords are better, then a followup PR should make all three classes consistent. Or drop keywords calls when creating `MrackHWDeviceElement`, that would also make them all aligned.